nox/virtualenv.py

```diff
@@ -182,6 +196,92 @@ def _resolved_interpreter(self):
         self._resolved = InterpreterNotFound(self.interpreter)
         raise self._resolved
 
+    @property
+    def _runtime_interpreter(self):
+        """
+        Returns the interpreter to use when shelling out, ``sys.executable``,
+        unless Nox was frozen with PyInstaller. In the later case, returns
+        system python interpreter matching the interpreter used when Nox was
+        frozen with PyInstaller or as low as Python 2.7, if found."""
+        if isinstance(self._runtime, Exception):
+            raise self._runtime
+
+        if self._runtime is not None:
+            return self._runtime
+
+        if not _FROZEN:
+            self._runtime = sys.executable
+            return self._runtime
+
+        def find_binary(major=None, minor=None):
```

Comment:
Please move this out as a helper function (I am big :-1: on closures, they are hard to reason about and test and the enclosed scope is not really accessible)

Reply:
ok